cardano-cli/src/Cardano/CLI/Shelley/Parsers.hs

```diff
@@ -320,6 +320,7 @@ pTransaction =
                                    <*> pTxTTL
                                    <*> pTxFee
                                    <*> pTxBodyFile Output
+                                   <*> some pCertificate
```

Comment:
```suggestion
                                   <*> many pCertificate
```
We use `some pTxIn` and `some pTxOut` because you do need at least one txin and txout, but it's perfectly ok to have zero certificates.

http://hackage.haskell.org/package/base-4.12.0.0/docs/Control-Applicative.html#v:some
> some :: f a -> f [a]
> One or more.
> many :: f a -> f [a]
> Zero or more.